securedrop/tests/test_journalist.py

```diff
@@ -678,6 +678,52 @@ def test_admin_add_user_integrity_error(self,
             " Please inform your administrator.",
             "error")
 
+    def test_logo_upload_with_valid_image_succeeds(self):
+        # Save original logo to restore after test run
+        logo_image_location = os.path.join(config.SECUREDROP_ROOT,
+                                           "static/i/logo.png")
+        with open(logo_image_location) as logo_file:
+            original_image = logo_file.read()
+
+        self._login_admin()
+
```

Comment:
this should be in a **try finally** so the image is restored even if the test fail